releases/v20.1.0-rc.1.md

```diff
@@ -0,0 +1,276 @@
+
+### Admin UI changes
+
+- The ALL filter on the Statements page now excludes internal statements. [#45646][#45646]
+- Tooltips showing statements and jobs are now limited in size for very long statements. [#46982][#46982]
+- The default timescale on metrics pages is now 10 minutes. Previously, the timescale defaulted to the age of the longest running node. [#46980][#46980]
+- Improved tooltips for existing capacity and storage metrics. [#46987][#46987]
+- Added analytics tracking for table sorts, searches and diagnostics activation on the Statements page, and navigation on the Statement Details page. [#47003][#47003]
+- The download link for statement diagnostics now points to the bundle zip file. [#47016][#47016]
+- Removed the Statements tab from the Databases > Table Details page. [#47102][#47102]
+- Cleaned up barcharts on the Statements page. [#47129][#47129]
+
+### Bug fixes
+
+- Fixed an Admin UI bug where sort columns were only being applied per-page instead of for the entire multi-page list of statements [#46978][#46978]
+- Fixed a performance bug where truncate would take 2*num columns + 2*num indexes round trips. This could lead to slow truncate performance in distributed clusters. [#46334][#46334]
+- It is no longer possible to inject stats within an explicit transaction. [#46567][#46567]
+- Casting a bit array to a bigger varbit array no longer adds extra 0 padding at the end. [#46532][#46532]
+- Fixed a bug where `pg_catalog.pg_indexes` showed the wrong index definition for inverted indexes. [#46527][#46527]
+- Fixed incorrect query results in some corner cases involving variance/stddev/corr. [#46436][#46436]
+- Fixed an internal error or incorrect evaluation of check constraints in certain cases involving `UPSERT` and foreign key checks. [#46409][#46409]
+- `cockroach debug zip` now properly collects heap profiles. [#46469][#46469]
+- The goroutine dump facility now functions properly when logging to files is disabled, e.g., via `--log-dir=""` or `--logtostderr`. [#46469][#46469]
+- Fixed an internal error that could occur during planning for some queries with a join and negative `LIMIT`. [#46440][#46440]
+- Fixed a bug where the vectorized engine could sometimes give an incorrect result when reading from interleaved parents or children. [#46456][#46456]
+- Fixed a bug where the vectorized engine would throw an internal error when executing a query that utilized an inverted index. [#46267][#46267]
+- Fixed a bug where operations on an index that contained a collated string in descending order would fail. [#46570][#46570]
+- Fixed a bug where `SET TIME ZONE` with a string prefixed with `UTC` or `GMT` and time zones that have colons had it's offset inverted the wrong way. [#46510][#46510]
```

Comment:
~This would be easier to read with some commas. Maybe something like "Fixed a bug with SET TIME ZONE, where a string prefixed with ... and time zones with colons, had its offset inverted ..."